docs/03_server/12_tooling/02_intellij-plugin.mdx

```diff
@@ -15,102 +15,182 @@ The Genesis Intellij Plugin enables you to run the full stack of a Genesis appli
+### Run Remap
 
-The Resource Daemon provides health information about your processes; it enables you to start and stop your processes via the Process view.
+With the Genesis Framework, [Remap](/operations/commands/remap) manages the database schema.
+Whenever tables are added, modified or removed, you will need to run remap to sync these changes.
+This includes when you run your project for the first time.
 
-To start the Resource daemon, press the **Start Resource Daemon** button.
+![Remap in the tool window](/img/intellij-plugin/tool-window-remap.png)
 
-Alternatively, if using the **startServer** button to start all of your processes at once, the Resource Daemon will be started for you.
+The Remap icon looks like a database with a green refresh arrows in the bottom right corner.
+Once you start Remap, at first it will run a sync, and then Remap will appear in the run Window:
 
-![Genesis Install](/img/plugin-resourcedaemon.png)
+![Running Remap](/img/intellij-plugin/remap-running.png)
 
-Once the resource daemon has started, you should see a list of your processes along with the status of each one.
+At this point, you will need enter `y` in the screen and hit `enter`.
+After which Remap will apply those changes and exit:
 
-### Starting processes
+![Running Remap](/img/intellij-plugin/remap-complete.png)
+
+:::important
+Remap cannot be run while other processes are running.
+We cannot use and update the database at the same time.
+However, if processes were running,
+the plugin will warn you and give you the option to stop all processes before proceeding.
+:::
 
-To start a process, click the **Start** button displayed for that process.
+### Loading Data
 
-You can also start all your processes at once using the **startServer** button in the tool window.
+If your project contains any required or sample data in CSV files, you load the data directly from IntelliJ.
+When you right-click on a CSV file, a number of selected CSV files,
+or on a folder containing CSV files, the "Import CSV(s) to Genesis" option will apppear:
 
-![Start Server](/img/plugin-startserver.png)
+![Import CSVs](/img/intellij-plugin/import-csvs.png)
 
-![Application Processes](/img/plugin-processes.png)
+Clicking on this option will import the data using [SendIt](/operations/commands/server-commands/?_highlight=sendit#sendit).
+Alternatively, you can also use SendIt directly from the [terminal](#using-the-terminal).
+
+## Using the Genesis IntelliJ Plugin
+
+Once you have completed the steps above, you are ready to start your application.
+The Genesis Tool Window will be the main entry point:
+
+![Genesis Tool Window](/img/intellij-plugin/genesis-tool-window.png)
+
+### Starting the front and backend
+
+To run your Genesis applicaiton locally, you can use these two buttons:
+
+![start buttons](/img/intellij-plugin/toolwindow-start-buttons.png)
+
+To start all services, click on the double green triangles.
+To start the UI, click on the globe with the green triangle.
```

Comment:
Might be better to move the UI bit to the section below

Maybe make `Starting the front and backend` into a level 2 heading, then have two level 3 headings one for server one for UI